components/sidebar/sidebar_header/sidebar_header.tsx

```diff
@@ -121,6 +121,7 @@ const SidebarHeader: React.FC<Props> = (props: Props): JSX.Element => {
                 id={'sidebar-header-container'}
             >
                 <OverlayTrigger
+
```

Comment:
Remove the space here

Reply:
when I save my file this space is automatically generated due to eslint.